### DISCUSSION TITLE
Issues with peer dependencies found - WARN

### DISCUSSION THREAD

**Discussion by ShueiYang** [pnpm/pnpm#6180] [Q&A] [answered]
Hi, i am testing pnpm for the 1st time for a small project with Next.js, however I get this warning message : 

```
 WARN  Issues with peer dependencies found
.
└─┬ eslint-config-next 13.2.3
  ├── ✕ missing peer typescript@>=3.3.1
  └─┬ @typescript-eslint/parser 5.54.0
    ├── ✕ missing peer typescript@"*"
    └─┬ @typescript-eslint/typescript-estree 5.54.0
      ├── ✕ missing peer typescript@"*"
      └─┬ tsutils 3.21.0
        └── ✕ missing peer typescript@">=2.8.0...
```
Well I know i can fix this if i install typescript for the peer dependencie from eslint-config-next, however for this project I don't want to use it, so is there a way to ignore / remove this warning message in the terminal ? 
Regards

**Comment by zkochan** [comment, marked as answer]
https://pnpm.io/package_json#pnpmpeerdependencyrules

in the root package.json add this:

```
{
  "pnpm": {
    "peerDependencyRules": {
      "ignoreMissing": ["typescript"]
    }
  }
}
```

**Reply by ShueiYang** [reply]
Damn I didn't see it on the doc -.- thanks :>
